i18n/en.json

```diff
@@ -438,6 +406,8 @@
 	"tool-authorship-desc": "Authorship attribution, measured by character count, excluding spaces",
 	"tool-autoedits": "Automated Edits",
 	"tool-autoedits-desc": "Analysis of semi-automated and non-automated contributions",
+	"tool-bash": "Random Quote",
+	"tool-bash-desc": "Get quotes from the IRC Quote Database",
```

Comment:
The tool is called Quote, so we should stick with that, I think, and instead fix the `xtPageTitle` and `xtSubtitle` in QuoteController::indexAction()